generators/workspace/templates/plain/metaData.json

```
@@ -0,0 +1,9 @@
+{
+    "buildId": "",
+    "name": "",
+    "version": "",
+    "repository": "",
+    "homepage": "",
+    "description": "",
+    "screenshot": ""
+   }
```

Missing new line at the end of the file.

```suggestion
   }

```